.eslintrc.yml

```diff
@@ -0,0 +1,14 @@
+---
+extends:
+  - 'eslint:recommended'
+  - 'plugin:node/recommended'
+  - prettier
```

Comment:
these linting templates are managed centrally by the code generator now, and we just took on a ton of work to make all the libraries consistent [here](https://github.com/googleapis/nodejs-kms/pull/309).

tldr; I don't think we should need to update any of this linting configuration manually.

Reply:
reverted